src/SurfaceFluxes.jl

```diff
@@ -683,10 +683,8 @@ function sensible_heat_flux(param_set, Ch, sc::Union{ValuesOnly, Coefficients},
     T_in = TD.air_temperature(thermo_params, ts_in(sc))
     T_sfc = TD.air_temperature(thermo_params, ts_sfc(sc))
     ΔT = T_in - T_sfc
-    hd_sfc = cp_d * (T_sfc - T_0) + R_d * T_0
     ΔΦ = grav * Δz(sc)
-    E = evaporation(param_set, sc, Ch)
-    return -ρ_sfc * Ch * windspeed(sc) * (cp_m * ΔT + ΔΦ) - (hd_sfc) * E
+    return -ρ_sfc * Ch * windspeed(sc) * (cp_m * ΔT + ΔΦ)
```

Comment:
I double-checked with Tapio about this. Here we assume that cp_m at the surface and the interior are the same (which is an ok assumption). To be more accurate we should use ΔDSE here as in the dycore paper (you can get cp_m at the surface from the surface thermo state). One caveat is that I don't know whether it is correct for land, we can leave a comment about land for now.

Reply:
Changed!